src/gsi/gsimod.F90

```diff
@@ -24,7 +24,7 @@ module gsimod
 
   use obsmod, only: doradaroneob,oneoblat,oneoblon,oneobheight,oneobvalue,oneobddiff,oneobradid,&
      radar_no_thinning,ens_hx_dbz_cut,static_gsi_nopcp_dbz,rmesh_dbz,&
-     rmesh_vr,zmesh_dbz,zmesh_vr,if_vterminal, if_model_dbz,if_vrobs_raw,&
+     rmesh_vr,zmesh_dbz,zmesh_vr,if_vterminal, if_model_dbz,if_vrobs_raw,if_use_w_vr,&
```

Comment:
Sorry, a picky comment here about the naming of "if_use_w_vr".
Without any further context, I would assume "if_use_w_vr" means using "w" and "vr".
suggest changing to a more verbose form, such as "if_use_w_in_vr", "if_use_w_for_vr", or "if_3D_wind_for_vr", etc. Thanks!

Reply:
@guoqing-noaa Good suggestion. @shoyokota Since it is easy changes, you may consider to include the variable name changes in your next PR. Because the PR already exceed the 6 weeks limit and I just finished a set of regression test, I'd like to merge this PR into "develop" today.

Reply:
@guoqing-noaa @ShunLiu-NOAA Thank you for your comments. Yes, it is easy. I will think about changing this name if I  create another PR related to the observation operator of radar radial wind.